source/Octopus.Tentacle.Tests/Kubernetes/KubernetesOrphanedPodCleanerTests.cs

```diff
@@ -72,28 +62,15 @@ public async Task OrphanedPodCleanedUpIfOver10MinutesHavePassed()
             await podService.Received().Delete(scriptTicket, Arg.Any<CancellationToken>());
         }
 
-        [TestCase("Succeeded", true)]
-        [TestCase("Failed", true)]
-        [TestCase("Running", false)]
+        [TestCase(PodState.Succeeded, true)]
```

Comment:
Add a case for `Pending` state?

Reply:
The `PodState` enum only has three states:
```csharp
    public enum PodState
    {
        Running,
        Succeeded,
        Failed
    }
```

Reply:
I see what you mean now - the Phase can have 5 different values.

Reply:
I've added additional tests for those phases 👍